packages/react/src/AnchoredOverlay/AnchoredOverlay.tsx

```diff
@@ -112,6 +117,7 @@ export const AnchoredOverlay: React.FC<React.PropsWithChildren<AnchoredOverlayPr
   alignmentOffset,
   anchorOffset,
   className,
+  preventOverflow = true,
```

Comment:
On by default, which means the overlay will NOT reflow unless `preventOverflow={false}` is explicitly set as a prop on the component. This is of course overriden by the `primer_react_overlay_overflow` FF introduced in https://github.com/primer/react/pull/5129/files#diff-228ff69d3a08b1a41b50995af899947232eb64943d74ffd7aefd6bdd80aac449R9